guide/07-working-with-bigdata/03-performing_big_data_analysis.ipynb

```diff
@@ -0,0 +1,956 @@
+{
```

Comment:
<p>Next we will specify which big data file share the GeoAnalyticss results will save to. If set to <code>None</code>, the <code>arcgis.env.output_datastore</code> will reset to default.  Allowed string values are: <code>spatiotemporal</code> or <code>relational</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1076/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p><code>None</code></p>